documentation/sphinx/source/release-notes/release-notes-7130.rst

```diff
@@ -0,0 +1,62 @@
+.. _release-notes:
+
+#############
+Release Notes
+#############
+
+71.3.0
+======
+
+Features
+--------
+- Added MonotonicTime field, based on system clock, to CommitDebug trace events, for accurate timing.
+- BlobGranule driven Backup + restore
+- Encryption at-Rest, supports per-tenant and/or per-cluster encryption semantics
+- External Vault support for per-tenant blob granule storage location
+- Improved APIs for creating and managing Blob Granule ranges
+- Updated tenants to be ID oriented. Tenants cannot be upgraded from previous releases.
+- Support for adding restored data clusters back to a metacluster and recovering a management cluster from the data clusters
+- Add support for locking tenants
+- Added a new function fdb_database_get_client_status providing a client-side connection status information in json format.
+- Added new database options to define defaults for transaction options report_conflicting_keys and used_during_commit_protection_disable.
+- Added a new network option retain_client_library_copies to avoid deleting the temporary library copies after completion of the process. This may be useful in various debugging and profiling scenarios.
+- Added a new network option trace_initialize_on_setup to enable client traces already on fdb_setup_network, so that traces do not get lost on client configuration issues
+- TraceEvents related to TLS handshake, new connections, and tenant access by authorization token are no longer subject to suppression or throttling, using an internal “AuditedEvent” TraceEvent classification
+- Usage of authorization token is logged as part of AuditedEvent, with 5-second suppression time window for duplicate entries (suppression time window is controlled by AUDIT_TIME_WINDOW flow knob)
+- Client-side arena blocks and packet buffers that held authorization tokens are zeroed out immediately before deallocation to minimize the likelihood of leaked tokens
+- Authorization token’s “aud” field may have singular string as value, instead of a singleton array (optional JWT standard)
+- RPC response to tenant-scoped get-key-location request now returns a key range clamped to the tenant keyspace to prevent leaking key range information of the normal keyspace or the other tenants
```

Comment:
```suggestion
- RPC response to tenant-scoped get-key-location requests now returns a key range clamped to the tenant keyspace to prevent leaking key range information of the normal keyspace to the other tenants.
```